src/java/boa/compiler/SymbolTable.java

```diff
@@ -188,7 +193,10 @@
 		// set functions
 		globalFunctions.addFunction("contains", new BoaFunction(new BoaBool(), new BoaType[] { new BoaSet(new BoaTypeVar("V")), new BoaTypeVar("V") }, "${0}.contains(${1})"));
 		globalFunctions.addFunction("add", new BoaFunction(new BoaAny(), new BoaType[] { new BoaSet(new BoaTypeVar("V")), new BoaTypeVar("V") }, "${0}.add(${1})"));
+		globalFunctions.addFunction("addAll", new BoaFunction(new BoaAny(), new BoaType[] { new BoaSet(new BoaTypeVar("V")), new BoaSet(new BoaTypeVar("V")) }, "${0}.addAll(${1})"));
```

Comment:
I'm a fan of union() because it does not mutate either argument.  You are given their union and have to store it somewhere.  Also as far as names go, 'union' *is* the general term for this and more widely known.  The 'addAll' is Java specific.